### DISCUSSION TITLE
Support for Snowflake

### DISCUSSION THREAD

**Discussion by CovantaDataManagement** [Budibase/budibase#2037] [Ideas]
Could you add data support for Snowflake?

**Comment by mike12345567** [comment]
Hi @CovantaDataManagement thanks for the feature request! Out of interest what way would you like to see Budibase support Snowflake? Would you see it being something similar to how we support other SQL data sources, simply allowing the ability to create SQL queries?

**Comment by CovantaDataManagement** [comment]
Definitely - we use snowflake to store the "gold" records through an operational data store.  It would be very useful to pull reference data and historical data from Snowflake.  I attempted to use the PostgreSQL driver to access but it did not work.

**Comment by CovantaDataManagement** [comment]
By the way - In my 30 years in IT - budibase is one of the best ideas I have ever come across.  Nice work!

**Reply by joebudi** [reply]
More than anything, this is what drives us. Thank you.
